File: chepa-desktop/ChepaMotos/ChepaMotos/Models/Mechanic.cs
using System.Text.Json.Serialization;

namespace ChepaMotos.Models;

public class Mechanic
{
    [JsonPropertyName("id")]
    public long Id { get; set; }

    [JsonPropertyName("name")]
    public string Name { get; set; } = string.Empty;

    [JsonPropertyName("is_active")]
    public bool IsActive { get; set; }
}

File: chepa-api/src/main/java/com/chepamotos/chepa_api/modelo/Mechanic.java
package com.chepamotos.chepa_api.modelo;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "mechanic")
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Mechanic {
    
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "mechanic_id")
    private Long mechanicId;
    
    @Column(name = "name", nullable = false, length = 100)
    private String name;
    
    @Column(name = "is_active", nullable = false)
    @Builder.Default
    private Boolean isActive = true;
}